scripts/make_next_milestone.py

```diff
@@ -32,6 +32,8 @@ def main():
         date = datetime.strptime(prev_milestone["due_on"], "%Y-%m-%dT%H:%M:%SZ") + timedelta(weeks=2)
         due_on = "%04d-%02d-%02dT%02d:%02d:%02dZ" % (date.year, date.month, date.day, date.hour, date.minute, date.second)
 
+    # After editing maintainers list, don't forget to update `SLACK_USERNAMES` GitHub secret.
+    # For mo info see `make_release_branch.py` file.
     release_managers = [
```

Comment:
nit: Maybe extract it into separate module together with `RELEASE_MANAGER_RE`?

Just not to have unexpected dependency between two scripts